packages/uilib/src/lib/plugins/plugin-store/plugin-store.svelte

```diff
@@ -0,0 +1,227 @@
+	requireDoc?: boolean
+	position?: MenuPosition
+	installed: boolean
+	official?: boolean
+}
+
+function storedPlugins(): Plugin[] {
+	return <Plugin[]>(
+		JSON.parse(
+			localStorage.getItem('plugins') ?? '[]',
+			(key, value) => value
+		)
+	)
+}
+
+function withoutContent<P extends Plugin>(plugin: P): P {
+	return { ...plugin, content: undefined }
+}
+
+function storePlugins(plugins: Array<Plugin>) {
+	localStorage.setItem('plugins', JSON.stringify(plugins.map(withoutContent)))
+}
+
+// Enables/disables plugin by toggling the "installed" property.
+function togglePluginState(plugin: Plugin, isEnabled: boolean) {
+	const currentPlugins = storedPlugins()
+	currentPlugins.find((it) => it.name === plugin.name).installed = isEnabled
+	storePlugins(currentPlugins)
+	plugins = currentPlugins
+	plugin.installed = isEnabled
+	notificationSnackbar.open()
+}
+
+// Completely removes plugin from local browser cache.
+function uninstallPlugin(plugin: Plugin) {
+	const currentPlugins = storedPlugins()
+	const updatedPlugins = currentPlugins.filter(
+		(it) => it.name !== plugin.name
+	)
+	storePlugins(updatedPlugins)
+	plugins = updatedPlugins
+}
+
+let plugins = storedPlugins()
+
+let showOnlyInstalled = false
+let searchFilter = ''
+
+$: filteredPlugins = plugins
+	.filter(
+		(plugin) => !showOnlyInstalled || plugin.installed === showOnlyInstalled
+	)
+	.filter((plugin) =>
+		plugin.name.toLowerCase().includes(searchFilter.toLowerCase())
+	)
+	.filter((plugin) => plugin.name !== 'PluginStore')
+	.filter((plugin) => plugin.name !== 'Plugin Store')
+
+// #endregion
+
+// #region UI
+let notificationSnackbar: Snackbar
+
+let currentPluginAnchor: Element
+let menus: Menu[]
+$: menus = filteredPlugins.map(() => null)
+$: menuStates = filteredPlugins.map(() => false)
+
+function openPluginMenu(e, index: number) {
+	currentPluginAnchor = e.currentTarget
+	menuStates = menuStates.map(() => false)
+	menuStates[index] = true
+}
+
+// #endregion
+</script>
+
+<Dialog bind:open={isOpen} 
+fullscreen
+surface$style="width: 850px; max-width: calc(100vw - 32px);"
+aria-labelledby="plugin-store-title" 
+aria-describedby="plugin-store-content">
+<Header>
+    <Title id="plugin-store-title">Plugin Store</Title>
+    <IconButton class="material-icons" action="close">
+        <IconClose/>
+    </IconButton>
+</Header>
+<Content id="plugin-store-content">
+<Theme>
+    <plugin-store>
+        <plugin-store-toolbar>
+            <Checkbox label="Only Installed" bind:checked={showOnlyInstalled}/>
+            <Textfield
+            label={"Search"}
+            variant={"outlined"}
+            bind:value={searchFilter}
+            />
+        </plugin-store-toolbar>
+        <plugin-store-items> 
+            {#each filteredPlugins as plugin, index}
+            <plugin-store-item style={`background: rgba(0, 0, 0, ${index % 2 === 0 ? "0" : "0.03"});`}>
+                <plugin-store-item-meta>
+                <span style="font-size: 0.85rem;">Author</span>
```

Comment:
No Author specified, just text "Author" is being shown

Reply:
This is intentional, it's just a placeholder for now. It needs to be retrieved from an external registry I'm working on #133.